code/datums/effects/xeno_strains/execute_tag.dm

```diff
@@ -0,0 +1,41 @@
+/datum/effects/execute_tag
+	effect_name = "execute tag"
+	duration = null
+	flags = DEL_ON_DEATH | INF_DURATION
+
+
+/datum/effects/execute_tag/New(atom/affected, mob/from = null, last_dmg_source = null, zone = "chest", ttl = 35)
+	. = ..()
+
+	QDEL_IN(src, ttl)
```

Comment:
Rather than constantly deleting and adding new effects every set_stat; why not just set stat either apply or remove the execute hud if the conditions are correct for it? It just creates extra garbage, and it makes the hud go away when they still meet the conditions for an execute.